src/alto-ui/Datagrid/components/DatagridCellInput/__tests__/DatagridCellInput.test.js

```diff
@@ -0,0 +1,376 @@
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
```suggestion
import { shallow } from 'enzyme';
```
Could you use shallow instead of mount? I think it's better to only render first component

Reply:
Shallow doesn't allow me to compare focused element.
This test just need `mount` instead `shallow`